bigquery/README.md

```diff
@@ -132,3 +137,16 @@ So the basic flow is:
 - In another window, run `DATASET_CONFIG_DIR=dataset_config/<my dataset> docker-compose up --build indexer`
 - Then if you want to run Data Explorer UI, don't include `elasticsearch` in
   `docker-compose up`: `docker-compose up --build -t 0 nginx_proxy ui apise kibana`
+
+#### Index in GKE
+
+This section applies if your Elasticsearch index is in GKE. For example,
+your index is static; you are working on a change to the UI or API server.
+
+```
+kubectl get pods
+kubectl port-forward ES_CLIENT 9200:9200
+git cherry-pick --no-commit b1561b796833d2f1f82ce7cce579b6262016f76c
```

Comment:
Is this referring to the commit one is working on? Or is this a local commit you have? I can't find this commit https://github.com/DataBiosphere/data-explorer-indexers/pull/118/commits/b1561b796833d2f1f82ce7cce579b6262016f76c